html/build.gradle.kts

```diff
@@ -91,9 +92,40 @@ subprojects {
                         }
                     }
                     scm {
-                        connection.set("scm:git://github.com/JetBrains/compose-jb.git")
-                        developerConnection.set("scm:git://github.com/JetBrains/compose-jb.git")
-                        url.set("https://github.com/jetbrains/compose-jb")
+                        connection.set("scm:git://github.com/JetBrains/compose-multiplatform.git")
+                        developerConnection.set("scm:git://github.com/JetBrains/compose-multiplatform.git")
+                        url.set("https://github.com/jetbrains/compose-multiplatform")
+                    }
+                }
+            }
+            publications {
+                val oldArtifactId = when (projectName) {
+                    "html-core" -> "web-core"
+                    "html-svg" -> "web-svg"
+                    "html-test-utils" -> "test-utils"
+                    "html-benchmark-core" -> "web-benchmark-core"
+                    "internal-html-core-runtime" -> "internal-web-core-runtime"
+                    "html-integration-core" -> "web-integration-core"
+                    "compose-compiler-integration" -> "compose-compiler-integration"
+                    "compose-compiler-integration-lib" -> "compose-compiler-integration-lib"
+                    else -> null
+                }
+
+                // TODO Remove this publishing in Compose 1.7. The package was migrated in 1.4.
+                if (oldArtifactId != null) {
+                    create<MavenPublication>("relocation") {
```

Comment:
Do we need to run it separately on CI? or does MavenPublication plugin takes care about it?

Reply:
No need, CI calls all publishings